### PR TITLE
Web Development Course

### PR DESCRIPTION
**Pull request by sainingo**
# Web Development Course _>_ First Capstone Project.
> Web Development Course
1. I use mobile-first approach
2. Configured Linters files
3. Design the UI with CSS & a bit of sass.
4. Used Javascript for functionalities such as rendering data dynamically
5. Used netlify for hosting

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                <button type="button" class="close-menu"><i class="fa fa-times fa-spin" aria-hidden="true"></i></button>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>CodeCamp Campaign</li>
+                </ul>   
+            </nav>
```
Comment (Hamzaoutdoors) [inline about.html]: - [ ] Good job implementing the navbar in your mobile version, but currently I can't navigate to the `home page` using the navbar menu as you can see in the screenshot below, Kindly fix this issue

navbar_issue

File: index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                <nav class="lower-nav">
+                    <ul>
+                        <li><a href="about.html">About</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li>CodeCamp Campaign</li>
+                    </ul>   
+                </nav>
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] Could you please make the navbar items in the center following the cross axis, you can achieve that by giving `justify-content: center;`

links

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @sainingo, I am **Hamza** your reviewer for this PR 😸 

_You did a great job, I really enjoyed surfing on your fancy website 😄_ 

There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ You used JavaScript to display dynamically the list of Participants 

Cheers and Happy coding!👏👏👏

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sainingo** [comment]
Hi @Hamzaoutdoors 
I made the changes required. Thanks

**Review by elyor-doniyorov — APPROVED** [review]
Hi @sainingo, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**